flask/boxwise_flask/auth_helper.py

```diff
@@ -169,9 +171,14 @@ def authorization_test(test_for, **kwargs):
                 401,
             )
 
-
-def test_base(requesting_user, base_id):
-    users_bases = requesting_user["base_ids"]
-    if base_id in users_bases:
-        return True
-    return False
+def user_can_access_base(requesting_user, base_id):
+    if "base_ids" in requesting_user:
+        users_bases = requesting_user["base_ids"]
+        if base_id in users_bases:
+            return SUCCESS
+    else:
+        #Log error - user doesnt have base ids
+        print("user doesnt have base ids cannot validate base_id " + str(base_id))
```

Comment:
We could start using Python logging right away, then later it'll be easier to set up the full logging infrastructure.

I usually do the following: At the top of a module I create a logger instance named after the module

```
import logging 
LOGGER = logging.getLogger(__name__)
```

Here we could then do `LOGGER.warning("user doesnt have...")`

Reply:
Also, why don't we raise an AuthError here?

Reply:
I considered it but adding all the logging felt like a seperate ticket. @HaGuesto @aerinsol maybe you guys can write one up?

Reply:
https://flask.palletsprojects.com/en/1.1.x/logging/

Reply:
can you @mc18g13 or @pylipp write a ticket. I'm not hundred per cent sure what you are suggesting that we should log. So, the ticket would be a great starting point for a discussion what we should log where. 

In the old code base we are actually using sentry and I would love to keep on working with it though it is mainly focused on error/exception logging. We already created [a ticket for the frontend](https://trello.com/c/1eah7fiJ), but not the backend.

If there is anything else besides errors/exceptions you want to log, let me know and why.

Reply:
will bring this up in next dev meeting